### PR TITLE
Travis CI and Coveralls

### PR DESCRIPTION
**Pull request by ronkorving**
This adds a config file for Travis and integration with coveralls.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -30,6 +17,27 @@ before_install:
   - make install
   - cd ..
 
+script:
```
Comment (stelcheck) [inline .travis.yml]: Can't we paralellize that? https://docs.travis-ci.com/user/customizing-the-build/#Build-Matrix

Comment (ronkorving) [inline .travis.yml]: Yes, but they won't share ZMQ installations. That takes up a lot of time.

Comment (ronkorving) [inline .travis.yml]: Also, it runs coveralls for all versions of Node then, which is a little silly.

But, it *is* a lot more elegant to split it up, so I reverted this.

**Comment by stelcheck** [comment]
Rebased an merged manually into master

**Pull request closed without merging**